### PR TITLE
Update automation-content to account for deltas in (vendor-furnished) CentOS 9 Stream bootstrap images

### PR DESCRIPTION
**Pull request by ferricoxide**
1. Updates `spel/scripts/amigen9-build.sh` and `spel/scripts/builder-prep-9.sh` to reflect correct, default repo-names.
1. Adds/corrects builder test-definitions

### CODE REVIEW

**Comment by ferricoxide** [comment]
CentOS 9 Stream logic vetted (from CodeBuild spel-commercial job-number 266):
~~~text
amazon-ebs.minimal-centos-7-hvm: ======================= 18 passed, 13 skipped in 30.64s ========================
amazon-ebs.minimal-centos-8stream-hvm: ======================= 17 passed, 14 skipped in 16.69s ========================
amazon-ebs.minimal-centos-9stream-hvm: ======================= 13 passed, 18 skipped in 20.06s ========================
amazon-ebs.minimal-ol-8-hvm: ======================= 17 passed, 14 skipped in 41.61s ========================
amazon-ebs.minimal-ol-9-hvm: ======================= 13 passed, 18 skipped in 14.28s ========================
amazon-ebs.minimal-rhel-7-hvm: ======================= 18 passed, 13 skipped in 41.20s ========================
amazon-ebs.minimal-rhel-8-hvm: ======================= 17 passed, 14 skipped in 14.35s ========================
amazon-ebs.minimal-rhel-9-hvm: ======================= 13 passed, 18 skipped in 16.04s ========================
~~~

**Review by lorengordon — DISMISSED** [review]
There's some kind of bad merge history here. The files in the changeset as depicted by GitHub show changes from the prior OL8 fixes.

**Comment by ferricoxide** [comment]
> There's some kind of bad merge history here. The files in the changeset as depicted by GitHub show changes from the prior OL8 fixes.

Bleah. After I noticed that the OL9 changes had been approved and merged, I rebased my branch from upstream/master to ensure I could run an "all builders" test (success-output noted in my prior comment). Not why the extra grumph.

It _shouldn't_ create double entries, though?

Any rate, I'll see if I can clean up my branch.

File: tests/minimal-linux.pkr.hcl [hunk 1]
```diff
@@ -13,11 +13,21 @@ variable "aws_source_ami_centos8stream_hvm" {
...
 variable "aws_source_ami_ol_8_hvm" {
   type    = string
   default = env("amazon_ebs_minimal_ol_8_hvm")
 }
 
+variable "aws_source_ami_ol_9_hvm" {
+  type    = string
+  default = env("amazon_ebs_minimal_ol_9_hvm")
+}
+
```
Comment (lorengordon) [inline tests/minimal-linux.pkr.hcl:30]: Still got spurious content here.

File: tests/minimal-linux.pkr.hcl [hunk 2]
```diff
@@ -113,11 +123,21 @@ build {
...
   source "amazon-ebs.base" {
     source_ami = var.aws_source_ami_ol_8_hvm
     name       = "minimal-ol-8-hvm"
   }
 
+  source "amazon-ebs.base" {
+    source_ami = var.aws_source_ami_ol_9_hvm
+    name       = "minimal-ol-9-hvm"
+  }
+
```
Comment (lorengordon) [inline tests/minimal-linux.pkr.hcl:140]: And also here...

**Pull request merged**